src/Index.js

```diff
@@ -783,6 +812,18 @@ Index.prototype.batchSynonyms = function(synonyms, opts, callback) {
   });
 };
 
+Index.prototype.replaceAllSynonyms = function(synonyms, opts, callback) {
```

Comment:
We probably should discuss if we should or not verify the `objectID` of the provided synonyms.